app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%# Intent for this file:
+
+Similar to the 3 spotlights we used in Mediaranker homepage #%>
```

Comment:
Maybe edit to the line below just to be explicit and clear

"Similar to the 3 spotlights we used in Mediaranker homepage using the _spotlight form" #%>